forem/forem
# Take Apple Authentication out of Beta

## What type of PR is this? (check all applicable)


## Description

Removing `:apple_auth` feature flag to make SIWA fully available.

## Related Tickets & Documents

https://github.com/forem/internalEngineering/issues/4

## QA Instructions, Screenshots, Recordings

1. Boot app locally, sign in with admin user and visit `/admin/customization/config`
2. Within the Authentication section you should be able to add Apple Authentication (without the need for any feature flag)
3. Add placeholder values for all 4 configs to enable the provider
   - Configuring this provider isn't trivial so there's no need to test it works locally unless you're able to
   - You can check the implementation works in DEV or other Forem instances
4. Visit `/enter` in an incognito window and ensure the provider is displayed

### UI accessibility concerns?

None - No changes have been made to the HTML

## Added tests?


## Added to documentation?


## [optional] Are there any post deployment tasks we need to perform?

All authentication changes are slightly more mission critical, in the sense that if it breaks it will avoid users from accessing their accounts. A good post deployment task would be to validate Apple and other authentication providers continue to work as expected in DEV (which gets deployed post merge).

## [optional] What gif best describes this PR or how it makes you feel?

![bad apple](https://media.giphy.com/media/4n0ptv9XAt1pS/giphy.gif)
